docs/vsphere-golden-image.md

```diff
@@ -33,36 +41,19 @@ In case the configuration file does not exist, VMware Tools installs an example
 directory as the location above, or you can download it from the
 [open-vm-tools](https://raw.githubusercontent.com/vmware/open-vm-tools/master/open-vm-tools/tools.conf) repository.
 
-Ensure the virtual machine has a valid IP address in vCenter. Refer to the *IP Addresses* section in the *Summary* tab 
-of the vSphere Web Client or run the `ipconfig` command in the Windows VM.
-
-Ensure the *VMware Tools* Windows service is running and will start on boot. Refer to [VMware Tools service 
-documentation](https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vm_admin.doc/GUID-0BD592B1-A300-4C09-808A-BB447FAE2C2A.html) 
-or run the following PowerShell command:
+#### Disable the network interface exclusion
 
-```powershell
-    Get-Service -Name VMTools | Select Status, StartType 
-```
-
-### Exclude the network interface in the VMware Tools configuration
-
-Enable the following entry in the VMware Tools configuration file to ensure that the cloned vNIC generated on 
-the Windows VM by the hybrid-overlay won't be ignored.
+Uncomment the `exclude-nics` option in the `tools.conf` file and clear any existing values, the value must be 
+empty (no value), to ensure that any vNIC generated on the Windows VM by the hybrid-overlay won't be excluded.
+So that, VMware Tools is able to report the virtual IP address in the vCenter.
```

Comment:
`won't be excluded, allowing VMware Tools to report the IP addresses in vCenter`